bootstrap/components/sources/eventhubs-source-topology-component.json

```diff
@@ -0,0 +1,93 @@
+{
+  "type": "SOURCE",
+  "name": "Event Hubs",
+  "subType": "EVENTHUBS",
+  "streamingEngine": "STORM",
+  "builtin": true,
+  "transformationClass": "com.hortonworks.streamline.streams.layout.storm.EventHubSpoutFluxComponent",
+  "fieldHintProviderClass": "com.hortonworks.streamline.streams.catalog.topology.component.bundle.impl.EventHubsSourceHintProvider",
+  "mavenDeps": "org.apache.storm:storm-eventhubs:STORM_VERSION^org.slf4j:slf4j-log4j12",
+  "topologyComponentUISpecification": {
+    "fields": [
+      {
+        "uiName": "Username",
+        "fieldName": "username",
+        "isOptional": false,
+        "tooltip": "The Event Hubs user name (policy name in Event Hubs Portal)",
+        "type": "string"
+      },
+      {
+        "uiName": "Password",
+        "fieldName": "password",
+        "isOptional": false,
+        "tooltip": "The Event Hubs password (shared access key in Event Hubs Portal)",
+        "type": "string",
+        "hint": "password"
+      },
+      {
+        "uiName": "Namespace",
+        "fieldName": "namespace",
+        "isOptional": false,
+        "tooltip": "The Event Hubs namespace",
+        "type": "string"
+      },
+      {
+        "uiName": "Entity path",
+        "fieldName": "entityPath",
+        "isOptional": false,
+        "tooltip": "The Event Hubs entity path",
+        "type": "string",
+        "hint": "schema"
```

Comment:
This will make UI query schema registry for schema. For kafka it uses topic name. Is that expected for this source as well? What will UI use to query schema registry?